internal/parser/scanner/ruleset/ruleset_default.go

```diff
@@ -71,7 +71,7 @@ func defaultKeywordsRule(s RuneScanner) (token.Type, bool) {
 	var buf bytes.Buffer
 	for {
 		next, ok := s.Lookahead()
-		if !ok || defaultWhitespaceDetector.Matches(next) {
+		if !ok || defaultWhitespaceDetector.Matches(next) || !defaultLiteral.Matches(next) {
```

Comment:
Is the whitespace check really neccessary any more?

Reply:
Ah yes, can be removed